[2024-06-27T19:02:09.838+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T19:02:09.876+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:01:18.385572+00:00 [queued]>
[2024-06-27T19:02:09.888+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:01:18.385572+00:00 [queued]>
[2024-06-27T19:02:09.889+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-27T19:02:09.906+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): run_silver_job> on 2024-06-27 19:01:18.385572+00:00
[2024-06-27T19:02:09.914+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=757) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-27T19:02:09.915+0000] {standard_task_runner.py:63} INFO - Started process 758 to run task
[2024-06-27T19:02:09.914+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Stage_1', 'run_silver_job', 'manual__2024-06-27T19:01:18.385572+00:00', '--job-id', '417', '--raw', '--subdir', 'DAGS_FOLDER/kafka_dag.py', '--cfg-path', '/tmp/tmp7fqplo5s']
[2024-06-27T19:02:09.916+0000] {standard_task_runner.py:91} INFO - Job 417: Subtask run_silver_job
[2024-06-27T19:02:09.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-27T19:02:09.967+0000] {task_command.py:426} INFO - Running <TaskInstance: Stage_1.run_silver_job manual__2024-06-27T19:01:18.385572+00:00 [running]> on host bafdbafde709
[2024-06-27T19:02:10.076+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Stage_1' AIRFLOW_CTX_TASK_ID='run_silver_job' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T19:01:18.385572+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-27T19:01:18.385572+00:00'
[2024-06-27T19:02:10.077+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T19:02:10.113+0000] {docker.py:366} INFO - Starting docker container from image bitnami/spark:latest
[2024-06-27T19:02:10.116+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-27T19:02:10.578+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.57 [0m[38;5;2mINFO [0m ==>
[2024-06-27T19:02:10.580+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.57 [0m[38;5;2mINFO [0m ==> [1mWelcome to the Bitnami spark container[0m
[2024-06-27T19:02:10.582+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.58 [0m[38;5;2mINFO [0m ==> Subscribe to project updates by watching [1mhttps://github.com/bitnami/containers[0m
[2024-06-27T19:02:10.584+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.58 [0m[38;5;2mINFO [0m ==> Submit issues and feature requests at [1mhttps://github.com/bitnami/containers/issues[0m
[2024-06-27T19:02:10.587+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.58 [0m[38;5;2mINFO [0m ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit [1mhttps://bitnami.com/enterprise[0m
[2024-06-27T19:02:10.589+0000] {docker.py:436} INFO - [38;5;6mspark [38;5;5m19:02:10.58 [0m[38;5;2mINFO [0m ==>
[2024-06-27T19:02:10.602+0000] {docker.py:436} INFO - 
[2024-06-27T19:02:13.432+0000] {docker.py:436} INFO - :: loading settings :: url = jar:file:/opt/bitnami/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-06-27T19:02:13.560+0000] {docker.py:436} INFO - Ivy Default Cache set to: /opt/bitnami/spark/.ivy2/cache
[2024-06-27T19:02:13.563+0000] {docker.py:436} INFO - The jars for the packages stored in: /opt/bitnami/spark/.ivy2/jars
[2024-06-27T19:02:13.573+0000] {docker.py:436} INFO - org.postgresql#postgresql added as a dependency
[2024-06-27T19:02:13.575+0000] {docker.py:436} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-60ebfe06-9f73-43df-83f7-8019ec4315df;1.0
[2024-06-27T19:02:13.577+0000] {docker.py:436} INFO - confs: [default]
[2024-06-27T19:02:16.110+0000] {docker.py:436} INFO - found org.postgresql#postgresql;42.2.2 in central
[2024-06-27T19:02:16.135+0000] {docker.py:436} INFO - downloading https://repo1.maven.org/maven2/org/postgresql/postgresql/42.2.2/postgresql-42.2.2.jar ...
[2024-06-27T19:02:16.296+0000] {docker.py:436} INFO - [SUCCESSFUL ] org.postgresql#postgresql;42.2.2!postgresql.jar(bundle) (172ms)
[2024-06-27T19:02:16.298+0000] {docker.py:436} INFO - :: resolution report :: resolve 2546ms :: artifacts dl 176ms
[2024-06-27T19:02:16.299+0000] {docker.py:436} INFO - :: modules in use:
[2024-06-27T19:02:16.300+0000] {docker.py:436} INFO - org.postgresql#postgresql;42.2.2 from central in [default]
	---------------------------------------------------------------------
	|                  |            modules            ||   artifacts   |
	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
	---------------------------------------------------------------------
	|      default     |   1   |   1   |   1   |   0   ||   1   |   1   |
	---------------------------------------------------------------------
[2024-06-27T19:02:16.304+0000] {docker.py:436} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-60ebfe06-9f73-43df-83f7-8019ec4315df
[2024-06-27T19:02:16.305+0000] {docker.py:436} INFO - confs: [default]
[2024-06-27T19:02:16.315+0000] {docker.py:436} INFO - 1 artifacts copied, 0 already retrieved (771kB/10ms)
[2024-06-27T19:02:16.686+0000] {docker.py:436} INFO - 24/06/27 19:02:16 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-06-27T19:02:18.073+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkContext: Running Spark version 3.5.1
[2024-06-27T19:02:18.074+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkContext: OS info Linux, 6.5.0-41-generic, amd64
[2024-06-27T19:02:18.075+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkContext: Java version 17.0.11
[2024-06-27T19:02:18.109+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceUtils: ==============================================================
[2024-06-27T19:02:18.111+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-06-27T19:02:18.111+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceUtils: ==============================================================
[2024-06-27T19:02:18.112+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkContext: Submitted application: PostgreSQL Connection with PySpark
[2024-06-27T19:02:18.138+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-06-27T19:02:18.149+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceProfile: Limiting resource is cpu
[2024-06-27T19:02:18.150+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-06-27T19:02:18.240+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SecurityManager: Changing view acls to: spark
[2024-06-27T19:02:18.241+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SecurityManager: Changing modify acls to: spark
[2024-06-27T19:02:18.242+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SecurityManager: Changing view acls groups to:
[2024-06-27T19:02:18.243+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SecurityManager: Changing modify acls groups to:
[2024-06-27T19:02:18.244+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: spark; groups with view permissions: EMPTY; users with modify permissions: spark; groups with modify permissions: EMPTY
[2024-06-27T19:02:18.601+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO Utils: Successfully started service 'sparkDriver' on port 38915.
[2024-06-27T19:02:18.644+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkEnv: Registering MapOutputTracker
[2024-06-27T19:02:18.708+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkEnv: Registering BlockManagerMaster
[2024-06-27T19:02:18.743+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-06-27T19:02:18.745+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-06-27T19:02:18.752+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-06-27T19:02:18.785+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-f6a0d46c-bd9b-4b52-856b-a31a233421c9
[2024-06-27T19:02:18.806+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-06-27T19:02:18.834+0000] {docker.py:436} INFO - 24/06/27 19:02:18 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-06-27T19:02:19.094+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-06-27T19:02:19.222+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-06-27T19:02:19.289+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar at spark://localhost:38915/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719514938062
[2024-06-27T19:02:19.294+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar at file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719514938062
[2024-06-27T19:02:19.296+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar to /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:02:19.438+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Starting executor ID driver on host localhost
[2024-06-27T19:02:19.439+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: OS info Linux, 6.5.0-41-generic, amd64
[2024-06-27T19:02:19.440+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Java version 17.0.11
[2024-06-27T19:02:19.452+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-06-27T19:02:19.454+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@565662c1 for default.
[2024-06-27T19:02:19.469+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Fetching file:///opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719514938062
[2024-06-27T19:02:19.491+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: /opt/bitnami/spark/.ivy2/jars/org.postgresql_postgresql-42.2.2.jar has been previously copied to /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:02:19.497+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Fetching spark://localhost:38915/jars/org.postgresql_postgresql-42.2.2.jar with timestamp 1719514938062
[2024-06-27T19:02:19.544+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO TransportClientFactory: Successfully created connection to localhost/127.0.0.1:38915 after 32 ms (0 ms spent in bootstraps)
[2024-06-27T19:02:19.553+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: Fetching spark://localhost:38915/jars/org.postgresql_postgresql-42.2.2.jar to /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/fetchFileTemp5785152487815853699.tmp
[2024-06-27T19:02:19.588+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/fetchFileTemp5785152487815853699.tmp has been previously copied to /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/org.postgresql_postgresql-42.2.2.jar
[2024-06-27T19:02:19.593+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Executor: Adding file:/tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/userFiles-f57718cb-f213-4698-93e5-1c7bb66e83de/org.postgresql_postgresql-42.2.2.jar to class loader default
[2024-06-27T19:02:19.608+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 34753.
[2024-06-27T19:02:19.610+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO NettyBlockTransferService: Server created on localhost:34753
[2024-06-27T19:02:19.611+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-06-27T19:02:19.622+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, localhost, 34753, None)
[2024-06-27T19:02:19.627+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO BlockManagerMasterEndpoint: Registering block manager localhost:34753 with 434.4 MiB RAM, BlockManagerId(driver, localhost, 34753, None)
[2024-06-27T19:02:19.631+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, localhost, 34753, None)
[2024-06-27T19:02:19.633+0000] {docker.py:436} INFO - 24/06/27 19:02:19 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, localhost, 34753, None)
[2024-06-27T19:02:20.294+0000] {docker.py:436} INFO - 2024-06-27 19:02:20,294:create_spark_session:INFO:Spark session created successfully
[2024-06-27T19:02:20.313+0000] {docker.py:436} INFO - 24/06/27 19:02:20 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-06-27T19:02:20.314+0000] {docker.py:436} INFO - 24/06/27 19:02:20 INFO SharedState: Warehouse path is 'file:/opt/bitnami/spark/spark-warehouse'.
[2024-06-27T19:02:23.075+0000] {docker.py:436} INFO - 2024-06-27 19:02:23,074:silver_load:INFO:Load silver table ...
[2024-06-27T19:02:24.444+0000] {docker.py:436} INFO - Traceback (most recent call last):
  File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 75, in <module>
[2024-06-27T19:02:24.445+0000] {docker.py:436} INFO - write_to_postgres()
  File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 71, in write_to_postgres
    silver_load(df,spark)
  File "/opt/bitnami/spark/scripts/spark_silver_layer.py", line 63, in silver_load
    df.write.jdbc(POSTGRES_URL, "silver_grocery_purchase", "append", properties=POSTGRES_PROPERTIES)
  File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 1984, in jdbc
[2024-06-27T19:02:24.446+0000] {docker.py:436} INFO - File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-06-27T19:02:24.446+0000] {docker.py:436} INFO - File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 185, in deco
[2024-06-27T19:02:24.447+0000] {docker.py:436} INFO - pyspark.errors.exceptions.captured.AnalysisException
[2024-06-27T19:02:24.455+0000] {docker.py:436} INFO - : Column item_name not found in schema Some(StructType(StructField(id,IntegerType,false),StructField(item_purchased,StringType,true),StructField(date_of_purchase,DateType,true),StructField(cost,DecimalType(10,2),true),StructField(quantity,IntegerType,true),StructField(purchased_by,StringType,true),StructField(payment_method,StringType,true),StructField(purchased_category,StringType,true))).
[2024-06-27T19:02:24.456+0000] {docker.py:436} INFO - 2024-06-27 19:02:24,454:close:INFO:Closing down clientserver connection
[2024-06-27T19:02:24.511+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO SparkContext: Invoking stop() from shutdown hook
[2024-06-27T19:02:24.512+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-06-27T19:02:24.525+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO SparkUI: Stopped Spark web UI at http://localhost:4040
[2024-06-27T19:02:24.540+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-06-27T19:02:24.572+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO MemoryStore: MemoryStore cleared
[2024-06-27T19:02:24.573+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO BlockManager: BlockManager stopped
[2024-06-27T19:02:24.581+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-06-27T19:02:24.584+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-06-27T19:02:24.592+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO SparkContext: Successfully stopped SparkContext
[2024-06-27T19:02:24.594+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO ShutdownHookManager: Shutdown hook called
[2024-06-27T19:02:24.596+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO ShutdownHookManager: Deleting directory /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7
[2024-06-27T19:02:24.599+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO ShutdownHookManager: Deleting directory /tmp/spark-a7c599ef-c49a-4199-b37e-f604a2d721f7/pyspark-c90baef9-0068-475f-9add-32c5ea01544a
[2024-06-27T19:02:24.603+0000] {docker.py:436} INFO - 24/06/27 19:02:24 INFO ShutdownHookManager: Deleting directory /tmp/spark-8f14d88a-58d5-4cd2-8e9b-151e61dc0d83
[2024-06-27T19:02:24.855+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T19:02:24.870+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 265, in _raise_for_status
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://docker-proxy:2375/v1.45/containers/create

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 371, in _run_image
    return self._run_image_with_mounts([*self.mounts, tmp_mount], add_tmp_variable=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 398, in _run_image_with_mounts
    self.container = self.cli.create_container(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py", line 439, in create_container
    return self.create_container_from_config(config, name, platform)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/container.py", line 456, in create_container_from_config
    return self._result(res, True)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 271, in _result
    self._raise_for_status(response)
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/api/client.py", line 267, in _raise_for_status
    raise create_api_error_from_http_exception(e) from e
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/docker/errors.py", line 39, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation) from e
docker.errors.APIError: 400 Client Error for http://docker-proxy:2375/v1.45/containers/create: Bad Request ("invalid mount config for type "bind": bind source path does not exist: /tmp/airflowtmp4kwyulm1")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 509, in execute
    return self._run_image()
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 380, in _run_image
    return self._run_image_with_mounts(self.mounts, add_tmp_variable=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/docker/operators/docker.py", line 444, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 1}
[2024-06-27T19:02:24.879+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Stage_1, task_id=run_silver_job, run_id=manual__2024-06-27T19:01:18.385572+00:00, execution_date=20240627T190118, start_date=20240627T190209, end_date=20240627T190224
[2024-06-27T19:02:24.901+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 417 for task run_silver_job (Docker container failed: {'StatusCode': 1}; 758)
[2024-06-27T19:02:24.948+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-27T19:02:24.981+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T19:02:24.984+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
